src/pages/components/number-input/style.mdx

```diff
@@ -76,15 +77,16 @@ The add and subtract icons can be found in the [icons](/guidelines/icons/library
 | `.bx--number`                      | border-bottom               | 1px      | –             |
 
 ## Sizes
-The height varies for each size variant and the the width varies based on content, layout, and design. 
+
+The height varies for each size variant and the the width varies based on
+content, layout, and design.
 
 | Element | Size        | Height (px/rem) |
 | ------- | ----------- | --------------- |
 | Field   | Small (sm)  | 32 / 2          |
 |         | Medium (md) | 40 / 2.5        |
 |         | Large (lg)  | 48 / 3          |
 
-
 <div className="image--fixed">
```

Comment:
You need to move this structure image and caption back to under the structure table as well.